spec/generators/dateGenerator.spec.js

```diff
@@ -0,0 +1,7 @@
+const date = require('../../generators/dateGenerator');
+
+describe('date generator', () => {
+  it('should return a date', () => {
```

Comment:
For your other tests, you used `test` instead of `it`. Not sure if you want to keep them consistent or not since Jest will read `test` and `it` but just something to note.